json/classic/src/test/java/ch/qos/logback/contrib/json/classic/JsonLayoutTest.java

```diff
@@ -48,11 +49,11 @@ public void addToJsonMap() throws Exception {
         jsonLayout.add("key2", false, "value2", map);
         jsonLayout.add("key3", true, null, map);
 
-        assertTrue(map.size() == 1);
-        assertTrue(map.containsKey("key1"));
+        assertThat(map.size(), is(1));
+        assertThat(map.containsKey("key1"), is(true));
         assertEquals(map.get("key1"), "value1");
-        assertFalse(map.containsKey("key2"));
-        assertFalse(map.containsKey("key3"));
+        assertThat(map.containsKey("key2"), is(false));
+        assertThat(map.containsKey("key3"), is(false));
```

Comment:
This comment wasn't addressed in your latest changes. Maybe you missed it. :-) If not, please share your concerns with it.